extra-utils/solaar/autobuild/beyond

```diff
@@ -0,0 +1,3 @@
+abinfo "Installing udev rules..."
+install -Dvm644 rules.d/42-logitech-unify-permissions.rules \
```

Comment:
Use absolute path here.

Reply:
Pwease appwove this pwull wequest senpai! 💍💍💍